### PR TITLE
SI-8368 respect user-supplied scala.usejavacp

### PR DESCRIPTION
**Pull request by adriaanm**
Now also works  when the option is -Dscala.usejavacp=false...

urghhhh

### CODE REVIEW

**Comment by gkossakowski** [comment]
I must say I'm clueless what this patch does but it's probably me not knowing bash well enough.

**Comment by adriaanm** [comment]
before it was doing x == "a", now it's doing x.startsWith("a")

File: src/compiler/scala/tools/ant/templates/tool-windows.tmpl [hunk 1]
```diff
@@ -49,7 +49,7 @@ if "%_TEST_PARAM:~0,2%"=="-J" (
 )
 
 if "%_TEST_PARAM:~0,2%"=="-D" (
-  if "%_TEST_PARAM%"=="-Dscala.usejavacp" (
+  if "%_TEST_PARAM:~0,17%"=="-Dscala.usejavacp" (
```
Comment (gkossakowski) [inline src/compiler/scala/tools/ant/templates/tool-windows.tmpl:53]: Shall we throw a comment saying 17 is length of "Dscala.usejavacp" or this is a common idiom that doesn't need any explanation?

**Comment by gkossakowski** [comment]
Sorry for being pedantic. I just fear I'll be the next guy to edit those scripts and I'm looking for trenches.

**Comment by adriaanm** [comment]
updated

**Comment by gkossakowski** [comment]
LGTM

Thanks. I'll let the tests run before merging.

**Pull request merged**